lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -160,6 +160,14 @@ def __post_init__(self, *args: Any, **kwargs: Any) -> None:
                     f"or '<Publisher>:<Offer>:<Sku>:<Version>'"
                 )
 
+    def _get_image_name(self) -> str:
```

Comment:
This method is called outside of itself, so do not add `_` before method name. `_` means it's a private method.